dask/tests/test_distributed.py

```diff
@@ -148,3 +149,17 @@ def test_futures_in_graph(loop):
             xxyy3 = delayed(add)(xxyy2, 10)
 
             assert xxyy3.compute(scheduler='dask.distributed') == ((1 + 1) + (2 + 2)) + 10
+
+
+def test_zarr_distributed_roundtrip(loop):
+    da = pytest.importorskip('dask.array')
+    zarr = pytest.importorskip('zarr')
+    assert_eq = da.utils.assert_eq
+    with cluster() as (s, [a, b]):
+        with Client(s['address'], loop=loop) as c:
+            with tmpdir() as d:
+                a = da.zeros((3, 3), chunks=(1, 1))
+                a.to_zarr(d)
+                a2 = da.from_zarr(d)
+                assert_eq(a, a2)
+                assert a2.chunks == a.chunks
```

Comment:
This looks good to me.  Thanks @jakirkham

Reply:
Thanks for checking :)